### DISCUSSION TITLE
[Desktop] Launch RustDesk On Startup

### DISCUSSION THREAD

**Discussion by trymeouteh** [rustdesk/rustdesk#6864] [General]
Have a setting to have Rustdesk be launched when the system starts up

Have another setting when launching Rustdesk at startup to launch it minimised to the tray.

**Comment by C-Duv** [comment]
Do you mean adding the ability for RustDesk to:

* starts when system/OS starts (before user logs in)
* starts minimized in tray (if it exists)

because it presently cannot do it?

or RustDesk can already do this and you simply want the GUI to be modified to add settings to configure such behaviours directly from the GUI?

**Comment by undee123** [comment]
ok, as the author of this thread seems to not respond anymore I will do it.
I'm exactly looking for these two options, did I miss something or it this not possible at all at the moment?

Option 2 seems to exist ("--tray") but I cannot connect to the host computer until the user does not double click on the tray icon and recall the main windows. Strange behaviour.


Edit: on nightly build there's the option "Security -> Only allow connection if Rustdesk windows is open", I did not have activated that option but Rustdesk anyway allows incoming connections only when windows is open. ???
